class-04-templates/pair-assignment/starter-code/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="styles/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+
+
+    <script id="blogArticles" type="text/x-handlebars-template">
+      {{#each articles}}
+      <section class="">{{title}}</section>
+      <section class="">{{category}}</section>
+      <section class="">{{{authorUrl}}}</section>
+      <section class="">{{publishedOn}}</section>
+      <section class="">{{{body}}}</section>
+      <section class="">{{pubDifference}}</section>
+      <hr />
```

Comment:
Use semantic html in your handlebars:
ex: <h1>{{title}}<h1>
etc.
This is needed because your styling is not working for headers, etc